### ISSUE TITLE
Preview the project

### ISSUE THREAD

**Issue by pel-daniel** [dmusican/Elegit#471]
Hi, 
Is there a way to preview the project without cloning it and installing all the dependencies ?
I am looking for something like a gif or a video in youtube.

**Comment by dmusican** [comment]
Thanks for the interest. We don't have a video (though we should, this is a good idea), but you can download a direct installer for your OS from elegit.org without having to grab the source code and build the thing. That should be a straightforward install -- if it isn't, we want to know. :)

I'm leaving this open as a way for us to track that we should add a screenshot and/or video somewhere.

**Comment by pel-daniel** [comment]
Thanks Dave, the installer was useful, I totally missed it in the page. I see you also updated the Readme to mention it.
I have done a git tutorial & cheatsheet that looks really similar to Elegit. Here are the links in case you want to check it out:
- [Cheatsheet](https://pel-daniel.github.io/git-init/cheatsheet.html)
- [Tutorial](https://pel-daniel.github.io/git-init/index.html)

**Comment by dmusican** [comment]
Cool! It looks even more similar than you know; we're about to add another column to show the index explicitly. I think one difference with the Elegit interface from the cheatsheet it showing the remote, and using push/fetch to manage that.

I really like the way you've used arrows for reset. I'm going to share that with our team. I think we should possibly consider adding that as well.

**Comment by pel-daniel** [comment]
Yes, I was planning to add remotes but I haven't had time to do it. One thing that I want to make explicit is which areas are in your computer and which out in "the cloud".

I was thinking that the interface can allow users to drag and drop from the different areas, maybe easing the use of source control for non-computer programmers like writers or designers.
